gr-digital/grc/digital_constellation.block.yml

```diff
@@ -49,9 +49,9 @@ templates:
         % else:
         self.${id} = ${id} = digital.constellation_${type}().base()
         % endif
-        % if str(soft_dec_lut).lower() == ''"auto"'' or str(soft_dec_lut).lower() == "''auto''":
+        % if str(soft_dec_lut).lower() == "auto" or str(soft_dec_lut).lower() == "auto":
```

Comment:
This doesn't look right. It's the same comparison twice in a row.

Reply:
Haha wow you're right